docs/Benchmarks.md

```diff
@@ -0,0 +1,94 @@
+# Benchmarks
+
+This document explains how to benchmark ASP.NET Core changes. It is intended for Microsoft employees.
+It describes how to trigger benchmarks using the Crank CLI and the Pull-Request bot.
+

```

Comment:
maybe here have a note that you're going to describe two alternatives -- local and PR -- below. they should choose which.

Reply:
OK